README.md

```diff
@@ -2,13 +2,21 @@
 ## 🚨 WARNING 🚨
 This package is only compatible with 64-bit installations of PHP.
 
-## Introduction
-
 ## Installation
 To add this library into your project, run:
 ```
 composer require moves/snowflake
 ```
 
 ## Usage
-
+Include this trait in your Laravel model by adding the following command to the model class:
+```
+use Moves\Snowflake\Traits\GeneratesSnowflakeId;
+```
+Then initialize the generator with the following:
+```
+self::bootGeneratesId(trait_type);
```

Comment:
The `bootGeneratesId` function is one that is called _automatically_ by Laravel, not manually by the developer.

Let's have a quick call to discuss this later - it shouldn't be a big deal to tweak this and get it working the way it's supposed to.